packages/core/src/tracing/instrumentation/messaging/bull.js

```diff
@@ -227,9 +212,21 @@ function instrumentedProcessJob(ctx, originalProcessJob, originalArgs) {
     const spanT = attributes.X_INSTANA_T;
     const spanP = attributes.X_INSTANA_S;
 
+    const parentSpan = cls.getCurrentSpan();
+
+    if (parentSpan && parentSpan.p === spanP && parentSpan.t === spanT) {
```

Comment:
Should we also maybe check parentSpan.n here? If I read this right, we want to allow starting a new span if the parent span is also a bull span that represents a concurrent Bull job invocation. 

If that interpretation is correct, would it might make sense to add `parentSpan.n !== 'bull'` to the list of checks?

Reply:
Good point. I think it makes sense.
I will add this verification!
Thanks